src/us/kbase/narrativemethodstore/db/test/NarrativeMethodDataTest.java

```diff
@@ -104,56 +105,71 @@ public void testGroupOneOk() throws Exception {
-				.getDynamicDropdownOptions();
-		
-		assertThat("incorrect data source", ddo.getDataSource(), is("custom"));
-		assertThat("incorrect serv func", ddo.getServiceFunction(),
-				is("taxonomy_re_api.search_taxa"));
-		assertThat("incorrect template", ddo.getDescriptionTemplate(),
-				is("<div>{{scientific_name}}</div>"));
-		assertThat("incorrect multiselect", ddo.getMultiselection(), is(1L));
-		assertThat("incorrect select id", ddo.getSelectionId(), is("scientific_name"));
-		assertThat("incorrect params",
-				ddo.getServiceParams()
-						.asClassInstance(new TypeReference<List<Map<String, Object>>>() {}),
-				is(Arrays.asList(ImmutableMap.of(
-						"search_text", "prefix:{{dynamic_dropdown_input}}",
-						"ns", "ncbi_taxonomy"))));
-		assertThat("incorrect serv ver", ddo.getServiceVersion(), is("dev"));
-		assertThat("incorrect sub path", ddo.getPathToSelectionItems(),
-				is(Arrays.asList("result", "0", "results")));
-		assertThat("incorrect query empty", ddo.getQueryOnEmptyInput(), is(0L));
-		assertThat("incorrect result index", ddo.getResultArrayIndex(), is(3L));
-	}
-	
-	@Test
-	public void dynamicDropDownDefaults() throws Exception {
-		final NarrativeMethodData nmd = load(11);
-		final DynamicDropdownOptions ddo = nmd.getMethodSpec().getParameters().get(0)
-				.getDynamicDropdownOptions();
-		
-		assertThat("incorrect data source", ddo.getDataSource(), is("custom"));
-		assertThat("incorrect serv func", ddo.getServiceFunction(),
-				is("taxonomy_re_api.search_taxa"));
-		assertThat("incorrect template", ddo.getDescriptionTemplate(),
-				is("<div>{{scientific_name}}</div>"));
-		assertThat("incorrect multiselect", ddo.getMultiselection(), is(0L));
-		assertThat("incorrect select id", ddo.getSelectionId(), is("scientific_name"));
-		assertThat("incorrect params",
-				ddo.getServiceParams()
-						.asClassInstance(new TypeReference<List<Map<String, Object>>>() {}),
-				is(Arrays.asList(ImmutableMap.of(
-						"search_text", "prefix:{{dynamic_dropdown_input}}",
-						"ns", "ncbi_taxonomy"))));
-		assertThat("incorrect serv ver", ddo.getServiceVersion(), nullValue());
-		assertThat("incorrect sub path", ddo.getPathToSelectionItems(), nullValue());
-		assertThat("incorrect query empty", ddo.getQueryOnEmptyInput(), is(1L));
-		assertThat("incorrect result index", ddo.getResultArrayIndex(), is(0L));
-	}
+    @Test
+    public void standardDropDown() throws Exception {
+        final NarrativeMethodData nmd = load(8);
+        // 3 has multiselection disabled
+        // 6 has it enabled
+
+        final DropdownOptions ddoSingle = nmd.getMethodSpec().getParameters().get(3)
+                .getDropdownOptions();
+        assertThat("incorrect multiselect", ddoSingle.getMultiselection(), is(0L));
+
+        final DropdownOptions ddoMulti = nmd.getMethodSpec().getParameters().get(6)
+                .getDropdownOptions();
+        assertThat("incorrect multiselect", ddoMulti.getMultiselection(), is(1L));
+    }
```

Comment:
git is dumb and doesn't realise that I inserted this code block and didn't change the next two tests.